src/code/z_fbdemo_circle.c

```diff
@@ -1,19 +1,134 @@
 #include "global.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo_circle/TransitionCircle_Start.s")
+Gfx D_801D0D00[] = { 0xE7000000, 0x00000000, 0xEF002CF0, 0x0342524D, 0xFA0000FF,
+                     0x00000001, 0xF9000000, 0x00000001, 0xDF000000, 0x00000000 };
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_fbdemo_circle/TransitionCircle_Init.s")
+// @bug TransitionCircle_Update may need to be casted? (void (*)(void* transition, s32 updateRate))?
+const TransitionInit TransitionCircle_InitVars = {
+    TransitionCircle_Init,   TransitionCircle_Destroy, TransitionCircle_Update,   TransitionCircle_Draw,
+    TransitionCircle_Start,  TransitionCircle_SetType, TransitionCircle_SetColor, NULL,
+    TransitionCircle_IsDone,
+};
```

Comment:
```suggestion
const TransitionInit TransitionCircle_InitVars = {
    TransitionCircle_Init,   TransitionCircle_Destroy, (void*)TransitionCircle_Update,   TransitionCircle_Draw,
    TransitionCircle_Start,  TransitionCircle_SetType, TransitionCircle_SetColor, NULL,
    TransitionCircle_IsDone,
};
```
I think Anghelo mentioned that casting as `void*` should remove the warning.